pkg/cri/config/config.go

```diff
@@ -392,11 +395,17 @@ func ValidatePluginConfig(ctx context.Context, c *PluginConfig) error {
 	useConfigPath := c.Registry.ConfigPath != ""
 	if len(c.Registry.Mirrors) > 0 {
 		if useConfigPath {
-			return errors.New("`mirrors` cannot be set when `config_path` is provided")
+			return warnings, errors.New("`mirrors` cannot be set when `config_path` is provided")
 		}
+		warnings = append(warnings, deprecation.CRIRegistryMirrors)
 		log.G(ctx).Warning("`mirrors` is deprecated, please use `config_path` instead")
 	}
 
+	if len(c.Registry.Configs) != 0 {
+		warnings = append(warnings, deprecation.CRIRegistryConfigs)
+		log.G(ctx).Warning("`configs` is deprecated, please use `config_path` instead")
```

Comment:
Bit orthogonal, and definitely not a blocker (I lost a better reference about this (:disappointed:) that had some great motivations / background on this), but generally "please" should be avoided in messages; here's some references (but perhaps there's better ones);

- https://ux.stackexchange.com/questions/25500/why-avoid-the-word-please-in-error-messages
- https://ux.stackexchange.com/questions/10853/should-an-interface-ever-say-please

Reply:
I've never considered that before!

I matched the existing messages which had "please".  Do you feel strongly enough that we should change this?

Reply:
No, not a blocker for this PR at all. I did notice the existing wording, so it made perfect sense to match that (consistency > being perfect), but it's something we could look at separately and check if we have more occurrences like that.